tests/unit/test_query_cli.py

```
@@ -0,0 +1,119 @@
+	def tearDown(self):
+		shutil.rmtree(self.temp_dir)
+
+	def _invoke(self, args):
+		from secator.cli import cli
+		captured = {}
+
+		def capture_send(report_self):
+			captured['results'] = report_self.data['results']
+
+		with mock.patch('secator.query.json.CONFIG') as mock_cfg, \
+				mock.patch('secator.report.Report.send', capture_send):
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)